[2024-06-24T00:00:14.920+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-24T00:00:14.938+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nasa_to_s3.transform_nasa_data manual__2024-06-23T23:48:08.347671+00:00 [queued]>
[2024-06-24T00:00:14.945+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nasa_to_s3.transform_nasa_data manual__2024-06-23T23:48:08.347671+00:00 [queued]>
[2024-06-24T00:00:14.946+0000] {taskinstance.py:2306} INFO - Starting attempt 4 of 7
[2024-06-24T00:00:14.958+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_nasa_data> on 2024-06-23 23:48:08.347671+00:00
[2024-06-24T00:00:14.965+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1648) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-24T00:00:14.966+0000] {standard_task_runner.py:63} INFO - Started process 1650 to run task
[2024-06-24T00:00:14.966+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nasa_to_s3', 'transform_nasa_data', 'manual__2024-06-23T23:48:08.347671+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/extractAPIdata.py', '--cfg-path', '/tmp/tmpuu767tn4']
[2024-06-24T00:00:14.968+0000] {standard_task_runner.py:91} INFO - Job 39: Subtask transform_nasa_data
[2024-06-24T00:00:15.002+0000] {task_command.py:426} INFO - Running <TaskInstance: nasa_to_s3.transform_nasa_data manual__2024-06-23T23:48:08.347671+00:00 [running]> on host e0fa33b4439a
[2024-06-24T00:00:15.076+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test' AIRFLOW_CTX_DAG_ID='nasa_to_s3' AIRFLOW_CTX_TASK_ID='transform_nasa_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-23T23:48:08.347671+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-23T23:48:08.347671+00:00'
[2024-06-24T00:00:15.077+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-24T00:00:15.121+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-24T00:00:15.121+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-24T00:00:15.129+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=nasa_to_s3, task_id=transform_nasa_data, run_id=manual__2024-06-23T23:48:08.347671+00:00, execution_date=20240623T234808, start_date=20240624T000014, end_date=20240624T000015
[2024-06-24T00:00:15.219+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-24T00:00:15.236+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-24T00:00:15.238+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
